aws/resource_aws_fsx_lustre_file_system.go

```diff
@@ -108,6 +108,26 @@ func resourceAwsFsxLustreFileSystem() *schema.Resource {
 					validation.StringMatch(regexp.MustCompile(`^[1-7]:([01]\d|2[0-3]):?([0-5]\d)$`), "must be in the format d:HH:MM"),
 				),
 			},
+			"deployment_type": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					fsx.LustreDeploymentTypeScratch1,
+					fsx.LustreDeploymentTypeScratch2,
+					fsx.LustreDeploymentTypePersistent1,
+				}, false),
+			},
+			"per_unit_storage_throughput": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				ForceNew: true,
+				ValidateFunc: validation.IntInSlice([]int{
+					50,
+					100,
+					200,
+				}),
+			},
```

Comment:
please add these fields to the read function.
something along the lines of:
```
d.Set("deployment_type", filesystem.LustreConfiguration.DeploymentType)
```

you would probably need to add default value of `fsx.LustreDeploymentTypeScratch1` to `deployment_type` as well.

Reply:
done, added a default and added the line you requested to the read function.